README.md

```diff
@@ -17,6 +18,9 @@ Hardware design for test device
     2. [Python Serial Monitor](#serial_monitor)
 5. [Hactar Setup](#hactar_setup)
 
+## License
+
+The license for information in this repository is in [LICENSE].  This license covers everything in the repository except for the `cmox` directory, which is covered by the license in [firmware/ui/dependencies/cmox/LICENSE].
```

Comment:
```suggestion
The license for information in this repository is in [LICENSE].  This license covers everything in the repository except for the directory `firmware/ui/dependencies/cmox`, which is covered by the license in [firmware/ui/dependencies/cmox/LICENSE].
```

Reply:
That change should make the links work right.  I think you'll need to commit the change to see the effect, though.